python/pylibcudf/pylibcudf/tests/test_interop.py

```diff
@@ -66,3 +67,10 @@ def test_decimal_other(data_type):
 
     arrow_type = plc.interop.to_arrow(data_type, precision=precision)
     assert arrow_type == pa.decimal128(precision, 0)
+
+
+def test_dlpack():
```

Comment:
For dlpack interop we should also test a few other types. Let's at least add a test for cupy arrays. Ideally I'd like to to something like PyTorch, but I don't want to make that a test dependency so let's maybe skip that for now. I would suggest at least numpy and cupy since those are the lightest-weight deps to install from [this list](https://dmlc.github.io/dlpack/latest/).